lib/config/presets/internal/monorepo.ts

```diff
@@ -41,7 +41,7 @@ const repoGroups = {
   ],
   'aws-cdk': 'https://github.com/aws/aws-cdk',
   'aws-lambda-powertools-typescript':
-    'https://github.com/awslabs/aws-lambda-powertools-typescript',
+    'https://github.com/aws-powertools/powertools-lambda-typescript',
```

Comment:
convert to array and add both urls, so it's working for old and new packages

Reply:
Done